src/app/static/src/app/components/plots/utils.ts

```diff
@@ -118,4 +118,19 @@ export const roundToContext = function (value: number, context: number) {
     const roundingNum = Math.pow(10, maxDecPl + 1);
 
     return Math.round(value * roundingNum) / roundingNum;
+};
+
+export const replaceAreaFilterOptionsWithCountryChildren = (filters: Filter[], areaFilterId: string) => {
```

Comment:
I don't understand what's going on here! The area filter gets passed to the component with options including the top level country but that's not actually what we want, so this method changes it to remove the top level? If I'm understanding that correctly, shouldn't the filter just be created with the desired options to begin with, i.e. in the getter?

Reply:
As for selectMultiple it depends on the plot - we do include the top level region for the Barchart. But again, I could put it in the getter for the maps but not the barchart..

Reply:
Ha, except this doesn't quite for the SAP getter, since the Choropleth needs the full option set so it can 'select' the country option if no options selected in the Filters component. Otherwise its default selection if the first region..